homeassistant/components/honeywell/climate.py

```diff
@@ -93,7 +94,13 @@ async def async_setup_entry(
 class HoneywellUSThermostat(ClimateEntity):
     """Representation of a Honeywell US Thermostat."""
 
-    def __init__(self, data, device, cool_away_temp, heat_away_temp):
+    def __init__(
+        self,
+        data: HoneywellData,
+        device: AIOSomecomfort.device.Device,
+        cool_away_temp: str | Any,
+        heat_away_temp: str | Any,
```

Comment:
We don't use these parameters. I'd remove them and the corresponding attributes.
```suggestion
```

Reply:
These are used here: https://github.com/home-assistant/core/blob/7e8c08106536c8ffef0cedd4cf73c16e9532ec99/homeassistant/components/honeywell/climate.py#L303

I've made some changes in this line of code in a later PR to address the issue where {mode} isn't cool or heat

Reply:
Ok, then just update the typing.

```suggestion
        cool_away_temp: str | None,
        heat_away_temp: str | None,
```

Reply:
Oops - lint said it was optional Any, so I went with that.